xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1484,7 +1484,9 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
 
     stream->source = STREAM_SOURCE_DEMUX;
     stream->pPrivate = pStream;
-    stream->flags = (CDemuxStream::EFlags)pStream->disposition;
```

Comment:
Doesn't you loose all other flags now? Additionally FLAG_DISP_ATTACHED_PIC = 0x0200 collides with AV_DISPOSITION_CLEAN_EFFECTS   0x0200